### DISCUSSION TITLE
Bug macOS: Ghost Panels Appear When Hiding and Showing Screen

### DISCUSSION THREAD

**Discussion by vitormalencar** [ghostty-org/ghostty#4188] [Feature Requests, Ideas] [answered]
When using Ghostty on macOS, ghost panels occasionally appear when hiding and showing the screen. These ghost panels persist until the Ghostty application is closed. A video demonstrating the issue is attached.

https://github.com/user-attachments/assets/4c2384f3-4e50-41d9-a0c1-ad5e29278c7b

### My current config 
```
cursor-style = block
window-decoration = true
background-opacity = 0.8
background-blur-radius = 20
macos-titlebar-style = "hidden"
mouse-hide-while-typing = true
font-size = 14
window-padding-x = 10
window-padding-y = 10

keybind = global:cmd+y=toggle_quick_terminal
keybind = global:cmd+\=toggle_visibility
```

### Steps to Reproduce:
Is mostly random, but I will add more steps if I'm able to.

Use Key Binds:

- Press the key binding to hide all windows, including Ghostty.
- Press the key binding to show all windows, including Ghostty.

### Observe Behavior:
- Occasionally, ghost panels appear after performing the hide and show actions.

### Environment:

-  Operating System: macOS Sonoma 14.6
-  Ghostty Version: 1.0.0 build 8469
-  Key Binds: `keybind = global:cmd+\=toggle_visibility`

Thank You! ❤️

I appreciate the time and effort you put into maintaining Ghostty. Please let me know if you need any further information to resolve this issue.

**Comment by mitchellh** [comment, marked as answer]
This was fixed in 1.0.1
